Src/Innovative.Blazor.Components/Components/SidePanel/SidePanelComponent.razor.css

```
@@ -112,6 +137,15 @@
     background-color: var(--inn-button-danger-hover-background-color, var(--inn-danger-dark));
     box-shadow: var(--inn-button-hover-shadow);
 }
+::deep .rz-button.rz-danger.rz-shade-default:focus {
```

Biome (2.1.2)

[error] Unexpected unknown pseudo-elements: deep

See MDN web docs for more details.
Use a known pseudo-elements instead, such as:


(lint/correctness/noUnknownPseudoElement)